### PR TITLE
Java.util.Locale support

### LINKED ISSUE #230: Add Go struct refer to java.util.Locale
**What would you like to be added**:

Add Go struct related to java.util.Locale

**Why is this needed**:

to support jdk classes.

### PR DESCRIPTION
**Pull request by zhangymPerson**
**What this PR does**:

**Which issue(s) this PR fixes**:

Fixes #230 

**Special notes for your reviewer**:add go struct Locale

**Does this PR introduce a user-facing change?**:

```release-note
add go struct Locale
```

### CODE REVIEW

File: java_util/locale.go [hunk 1]
```diff
@@ -0,0 +1,240 @@
...
+// default value is ROOT
+func GetLocale(localeEnum LocaleEnum) *Locale {
+	// Useful constant for language.
+	var ROOT = Locale{
+		id:     22,
+		Lang:   "",
+		County: "",
+	}
+	for _, locale := range locales {
+		if localeEnum == CHINA || localeEnum == PRC {
```
Comment (wongoo) [inline java_util/locale.go:213]: check first at the beginning of the func

File: java_util/locale.go [hunk 2]
```diff
@@ -0,0 +1,240 @@
...
+	}
+	locales = append(locales, ENGLISH, FRENCH, GERMAN, ITALIAN, JAPANESE, KOREAN, CHINESE, SIMPLIFIED_CHINESE, TRADITIONAL_CHINESE, FRANCE,
+		GERMANY, ITALY, JAPAN, KOREA, CHINA, PRC, TAIWAN, UK, US, CANADA, CANADA_FRENCH, ROOT)
+}
+
+// GetLocale is use enum value get Locale,
+// default value is ROOT
+func GetLocale(localeEnum LocaleEnum) *Locale {
+	// Useful constant for language.
+	var ROOT = Locale{
```
Comment (wongoo) [inline java_util/locale.go:207]: define root as const？

File: java_util/locale.go [hunk 3]
```diff
@@ -0,0 +1,240 @@
...
+		id:     22,
+		Lang:   "",
+		County: "",
+	}
+	for _, locale := range locales {
+		if localeHandler.Value == locale.String() {
+			return &locale
+		}
+	}
+	return &ROOT
```
Comment (wongoo) [inline java_util/locale.go:239]: define ROOT as const

File: java_util/locale.go [hunk 4]
```diff
@@ -0,0 +1,240 @@
...
+func (LocaleHandle) JavaClassName() string {
+	return "com.alibaba.com.caucho.hessian.io.LocaleHandle"
+}
+
+var locales []Locale = make([]Locale, 22, 22)
+
+// init java.util.Locale static object
+func init() {
+	// Useful constant for language.
+	var ENGLISH = Locale{
```
Comment (wongoo) [inline java_util/locale.go:80]: define these locales as consts

Comment (zhangymPerson) [reply java_util/locale.go:81]: ok,i know ,i will revise these questions

**Review by wongoo — APPROVED** [review]
LGTM

**Pull request merged**